coder/main.go

```diff
@@ -170,9 +187,36 @@ func Run(s *discordgo.Session, m *discordgo.MessageCreate) error {
 	//cli, err := client.NewEnvClient()
 	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
 	defer cancel()
+	
+	//Has attachments files
+	if(len(m.Attachments) > 0){
+		lang:="undefined";
+
+		
+		isPy,_ :=regexp.MatchString(`\.py?`, m.Attachments[0].URL)
+		isJs, _ :=regexp.MatchString(`\.js?`,m.Attachments[0].URL)
```

Comment:
I think it'd be cleaner to use `filepath.Ext()` on these: https://golang.org/pkg/path/filepath/#Ext

And then use parseLang as described in the below comment.

Reply:
Using regex was considered to allow for users to send a .py.txt file and for it to still be a valid source file. filepath.Ext() would only consider .txt

filepath.Ext(strings.TrimSuffix(m.Attachments[0].URL,".txt"))
is a working replacement

Reply:
I think the user should upload a `.py` file instead of a `.py.txt` file.